tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -102,8 +102,22 @@ def bulk_create_records(unsaved_records, line_number, header_count, datafile, ba
             for document, records in unsaved_records.items():
                 num_expected_db_records += len(records)
                 created_objs = document.Django.model.objects.bulk_create(records)
-                num_elastic_records_created += document.update(created_objs)[0]
                 num_db_records_created += len(created_objs)
+
+                try:
+                    num_elastic_records_created += document.update(created_objs)[0]
+                except BulkIndexError as e:
+                    logger.error(f"Encountered error while indexing datafile documents: {e}")
```

Comment:
it will be useful to add information about the failed record such as file, line_number at least for debugging purposes